src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -13,16 +13,12 @@ function isPreview() {
 $(document).ready(function() {
 
     /**
-     * Handles Keyup and Keydown on Text question to display response length
+     * Handles input on Text question to display response length
      */
-    $('textarea[id^="responsetext-"]').keyup(function() {
-        updateTextQuestionCharCount(this.id, $(this).data('lengthtextid'));
+    $('textarea[id^="responsetext-"]').on('input', function() {
+        updateTextQuestionWordsCount(this.id, $(this).data('length-text-id'), $(this).data('recommended-text'));
     });
-
-    $('textarea[id^="responsetext-"]').keydown(function() {
-        updateTextQuestionCharCount(this.id, $(this).data('lengthtextid'));
-    });
-
+    
     /**
      * Triggering keyup event for all text question type textfields, to call
```

Comment:
Will need you to change `$('textarea[id^="responsetext-"]').keyup();` to `$('textarea[id^="responsetext-"]').trigger('input');`, do edit the comment to `input` event instead of `keyup` event also.